exonum-java-binding/core/rust/src/utils/jni_cache.rs

```diff
@@ -195,7 +195,7 @@ fn get_class(env: &JNIEnv, class: &str) -> Option<GlobalRef> {
     let class = env
         .find_class(class)
         .unwrap_or_else(|_| panic!("Class {} not found", class));
-    Some(env.new_global_ref(class.into()).unwrap())
+    Some(env.new_global_ref(class).unwrap())
```

Comment:
Are there any other places where we can rely on implicit conversion into JObject?

Reply:
I remove the most obvious ones (which I could find using text search for `into`). It's possible that there are more subtle cases in our codebase

Reply:
Perfect, we can update the rest when find them.